docs/source/user.rst

```diff
@@ -0,0 +1,199 @@
+..  _user:
+
+Here the simple examples cover the basics of creating messaging applications using kiwipy over RabbitMQ.
+
+The parameters section will focus on some of the important parameters that initialize the communicator and their meaning.
+
+***************
+Simple examples
+***************
+
+RPC
+---
+
+The client:
+
+.. code-block:: python
```

Comment:
in spinx, you can use [includes](https://www.sphinx-doc.org/en/1.5/markup/code.html#includes) to avoid code duplication

Reply:
if you want to avoid the __future__ statements, you can start including from a certain line onwards (something like `:lines: 2-` )